src/DatePicker/index.js

```diff
@@ -68,7 +70,7 @@ export default class DatePicker {
         <button type="button" class="date-picker__toggle" aria-label="Toggle Date Picker">Toggle Date Picker</button>
 
         <div class="date-picker__calendar" style="display: none">
-          <p class="calendar__header" role="heading"></p>
+          <p class="calendar__header" role="heading" aria-live="assertive"></p>
```

Comment:
How might I get this to be announced anytime the datepicker is opened? Right now it's only read if you change months, or directly interact with it.